docs/peggy/docs/glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
```

Comment:
- Missing EthBridgeClaim
- ProphecyClaim is sif->evm only, 

I'll Add those changes